modules/openy_features/openy_location/modules/openy_loc_facility/config/install/core.entity_form_display.node.facility.default.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
     - node.type.facility
   module:
     - address
+    - content_moderation
```

Comment:
No need to add content moderation for facility. This is the super rare case when Ys adding facilities.